### DISCUSSION TITLE
Possible implements to SmartBuff??

### DISCUSSION THREAD

**Discussion by Angelmeat** [Softrix/SmartBuff-Retail#2] [Ideas]
Hi there, really love this addon, but I really miss the option to add toys and abilities in an easy way.
Therefore I'm curious if it would be possible to "borrow" some code from the addon called: QuestClicks by Krebbster.
In that addon there is a menu that allows you to just drag the icon straight in to add it in a inclusion list, and also the other way around, if you want to remove something that you don't want there.
If this is possible to implement to SmartBuff , that would be a huge success!

Thanx in advance! Cheers!

**Comment by ynazar1** [comment]
Heya I didn't even see this before... That's not possible with how SB implements and calculates buff lists and will require full rewrite of the addon logic.
